### DISCUSSION TITLE
Is there a way to add ignoreMissing for peer in only one workspace package?

### DISCUSSION THREAD

**Discussion by anantoghosh** [pnpm/pnpm#4502] [Q&A]
Currently if I add
```
"pnpm": {
    "peerDependencyRules": {
      "ignoreMissing": [
        "@babel/core"
      ]
    }
  }
```
in a **workspace package**, this is ignored and warnings are displayed.
It, however, works when added in the root package.json, but then it ignores the dependency warning for every workspace package.

Is there a way to ignore a peer dependency for only some packages in a workspace?

**Comment by vjpr** [comment]
I need this too because of emotion. For now you should use the pnpmfile to remove the peer dep when the package is resolved.

**Comment by i7eo** [comment]
I need this too !

**Comment by fkrauthan** [comment]
I would like to use this too. I have a pnpm workspace project with frontend and backend where I have different peer dependency rules.

**Comment by cllemon** [comment]
much needed !
